src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,69 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String PARAMETER_0_SUPPLY = "supply";
+    private static final String PARAMETER_1_BUY = "buy";
+    private static final String PARAMETER_2_RESULT = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] incomeData = getIncomeData(fromFileName);
+        int[] reportNumbers = getReportNumbers(incomeData);
+        writeReportToFile(toFileName, reportNumbers);
+    }
+
+    private String[] getIncomeData(String fromFileName) {
+        StringBuilder stringBuilder = new StringBuilder();
+        String[] resultData;
+        File incomeData = new File(fromFileName);
+        try (BufferedReader reader = new BufferedReader(new FileReader(incomeData))) {
+            String lineValue = reader.readLine();
+            while (lineValue != null) {
+                stringBuilder.append(lineValue).append(",");
+                System.out.println(lineValue);
+                lineValue = reader.readLine();
+            }
+            resultData = stringBuilder.toString().split(",");
```

Comment:
It's up to you, but returning a data as one string would make processing in the next method easier (we would just break by line separator and deal with each line separately)